cetmix_tower_server/models/cx_tower_plan_line.py

```diff
@@ -10,12 +10,15 @@
 
 class CxTowerPlanLine(models.Model):
     _name = "cx.tower.plan.line"
+    _inherit = [
+        "cx.tower.reference.mixin",
+    ]
     _order = "sequence, plan_id"
     _description = "Cetmix Tower Flight Plan Line"
 
     active = fields.Boolean(related="plan_id.active", readonly=True)
     sequence = fields.Integer(default=10)
-    name = fields.Char(related="command_id.name", readonly=True)
+    name = fields.Char(related="command_id.name", store=True)
```

Comment:
Probably we can try to leave it as is (non-stored, readonly) and see if it works.